addons/website_links/static/src/js/website_links.js

```diff
@@ -533,6 +535,26 @@ publicWidget.registry.websiteLinks = publicWidget.Widget.extend({
     },
 });
 
+popupWidget.include({
+    /**
+     * @override
+     */
+    _onHideModal: function (){
```

Comment:
```suggestion
    _onHideModal: function () {
```
Can't miss it with a proper configured linter
![image](https://github.com/odoo/odoo/assets/30048408/04b9d64b-3609-4d9d-aea1-5b42f6f90fd1)